bpf/flows.c

```diff
@@ -140,16 +140,15 @@ static inline int flow_monitor(struct __sk_buff *skb, u8 direction) {
         update_existing_flow(aggregate_flow, &pkt, len, filter_sampling);
     } else {
         // Key does not exist in the map, and will need to create a new entry.
-        flow_metrics new_flow = {
```

Comment:
we use designated init in many places, should we change all of them?

Reply:
not for flow_metrics , I think the old version of verifier wasn't happy with the embedded spin lock struct w/o initializtion